s3/pkg/datastore/alibaba/alibaba.go

```diff
@@ -314,8 +330,8 @@ func (ad *OSSAdapter) CompleteMultipartUpload(ctx context.Context, multipartUplo
 
 func (ad *OSSAdapter) AbortMultipartUpload(ctx context.Context, multipartUpload *pb.MultipartUpload) error {
 
-	bucket := ad.backend.BucketName
-	newObjectKey := multipartUpload.Bucket + "/" + multipartUpload.Key
+	bucket := multipartUpload.Bucket
```

Comment:
How is the failure of object upload handled

Reply:
It's just object parameter. so error handling is not required.